.github/workflows/list_likelihoods.yml

```
@@ -0,0 +1,28 @@
+name: List Public Likelihoods
+
+on:
+  push:
```

```suggestion
  push:
    branches: ["master"]
```

Should also probably have this run on pushes to `master` (PR merges) generally rather than every push.